### ISSUE TITLE
xFailOverCluster: Rename when this module is becoming HQRM

### ISSUE THREAD

**Issue by johlju** [PowerShell/xFailOverCluster#69]
**Details of the scenario you tried and the problem that is occurring:**
This might be getting ahead of things. But, when we get to the point that this module is HQRM and being renamed from experimental naming, I suggest we name it FailoverClustersDsc to match the Failover Clustering PowerShell module.

**The DSC configuration that is using the resource (as detailed as possible):**
n/a

**Version of the Operating System and PowerShell the DSC Target Node is running:**
n/a

**Version of the DSC module you're using, or 'dev' if you're using current dev branch:**
Dev

**Comment by bgelens** [comment]
I don't like it being plural. FailOverClusterDsc would be my preference

**Comment by johlju** [comment]
I agree. The PowerShell module for Failover Clustering is dubbed "FailoverClusters", so that's where I got it. :) With lowercase 'o' in 'over' and being plural. But agree it should be singular.

**Comment by bgelens** [comment]
What are you talking about ;) Fixed typo!

**Comment by johlju** [comment]
I updated the title of this issue, and change the label to help wanted, so that we can start work on this issue. 
Does anyone like to run with this issue? 😄
